build.js

```diff
@@ -97,133 +75,114 @@ const getCustomTestAPI = (name, member) => {
   return test;
 };
 
+const getCustomSubtestsAPI = (name) => {
+  const subtests = {};
+
+  if (name in customTests.api) {
+    const testbase = customTests.api[name].__base || '';
+    if ('__additional' in customTests.api[name]) {
+      for (
```

Comment:
I've filed https://github.com/foolip/mdn-bcd-collector/issues/432 to undo weird Prettier formatting like this.